app/js/components/LabOrdersList.jsx

```diff
@@ -154,6 +174,31 @@ export class LabOrdersList extends PureComponent {
     dispatch(filtersAction.setLabOrdersListFilters(newFilters));
   }
 
+  async handleCancel(order) {
+    const { currentProvider, dispatch, labResultsTestOrderType } = this.props;
+    const cancelConfirmation = await swal("Are you sure you would like to cancel this order ?", {
+      buttons: {
+        YES: "YES",
+        NO: 'NO',
+      },
+    });
+    if (cancelConfirmation === "YES") {
+      const cancelledOrder = {
+        ...order,
+        orderer: currentProvider.uuid,
+        previousOrder: order.uuid,
+      };
+
+      const payload = {
+        encounterType: cancelledOrder.encounter.uuid,
+        orders: [cancelledOrder],
+        patient: cancelledOrder.patient.uuid,
+      };
+
+      // dispatch(updateOrder(payload));
```

Comment:
Why isn't updateOrder dispatched?

Reply:
thanks, @paulvazbad, but this PR is still a work-in-progress. I stated on the title that I was pausing to figure out which rest endpoint to actually do a post request to, hence the reason for commenting out the dispatch.